radixdlt-java/src/test/java/com/radixdlt/client/core/crypto/ECKeyPairTest.java

```diff
@@ -1,18 +1,20 @@
 package com.radixdlt.client.core.crypto;
 
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-
 import org.junit.Test;
 
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
 public class ECKeyPairTest {
-	@Test
-	public void decryptBadEncryptedDataWithGoodEncryptedPrivateKeyShouldThrowCryptoException() {
-		ECKeyPair keyPair = ECKeyPairGenerator.newInstance().generateKeyPair();
-		ECKeyPair privateKey = ECKeyPairGenerator.newInstance().generateKeyPair();
 
-		EncryptedPrivateKey encryptedPrivateKey = privateKey.encryptPrivateKey(keyPair.getPublicKey());
+    @Test
+    public void decrypt_bad_encrypted_data_with_good_encrypted_private_key__should_throw_CryptoException() {
```

Comment:
I don't believe this passes checkstyle after running `./gradlew build`. I've tried updating `checkstyle.xml` in the past but ran into problems then got lazy :) Can you either update checkstyle.xml or fix the method signature so that `./gradlew build` passes?